Items/07_Items.txt

```diff
@@ -205,6 +205,80 @@ Road Flare ($15):
 light for 5m in every direction for about an hour. Burns in rain, mud, even 
 burns underwater. Don't touch the end or you'll take 10 fire damage.
 
+Datapad (130$):
+	A handheld device with a screen and keypad. Used to record
+information up to 1TB. 
+
+Binoculars (90$):
+	A bifocal optical instrument with a lens for each eye, 
+used for viewing distant objects up to 2km. 
+
+Industrial Paint (35$):
+	A metal canister of wear-resistant paint. Can be used 
+with a brush or similar implement to cover up to 20 square meters of
+flooring, wall or other surface. Paint comes in the following colors.
+
+* Black
+* White
+* Red
+* Blue
+* Green
+* Yellow
+* Orange
+* Purple
+* Brown
+
+Adhesive Foam (45$):
+	A spray can with a nozzle on the end that sprays a chemical
+solution in either an area or a thin line. As the solution dries, 
+it expands and hardens to accomodate the space given. Can be used
+to insulate an area, temporarily patch up a damaged wall, 
+seal ducts, etc. One can has 20 square meter's worth of usage.
```

Comment:
n-b: "One can contains" and "worth of usage" could be written "worth of foam"
block: Cubic meters may be a more useful unit of measurement.